src/io/flutter/run/LauncherState.java

```diff
@@ -50,28 +50,28 @@
 
 /**
  * Launches a flutter app, showing it in the console.
- *
- * <p>Normally creates a debugging session, which is needed for hot reload.
+ * <p>
+ * Normally creates a debugging session, which is needed for hot reload.
  */
-public class Launcher extends CommandLineState {
+public class LauncherState extends CommandLineState {
```

Comment:
Hmm, renaming this seems unfortunate due to the inner interfaces. LauncherState.RunConfig etc get longer.